acceptance_tests/01-count-pe1.py

```diff
@@ -1,5 +0,0 @@
-from bluesky.plans import count
```

Comment:
Please restore all of the acceptance tests.

Reply:
What is the purpose of acceptance_tests ?